packages/shared/src/components/cards/common/CardCover.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactElement, useState } from 'react';
+import classNames from 'classnames';
+import { ImageProps, ImageType } from '../../image/Image';
+import VideoImage, { VideoImageProps } from '../../image/VideoImage';
+import ConditionalWrapper from '../../ConditionalWrapper';
+import { CardImage } from '../Card';
+import { CardCoverShare } from './CardCoverShare';
+import { CommonCardCoverProps } from '../common';
+import {
+  upvoteMutationKey,
+  useMutationSubscription,
+  UseVotePostMutationProps,
+} from '../../../hooks';
+import { useActiveFeedNameContext } from '../../../contexts';
+import { useFeatureIsOn } from '../../GrowthBookProvider';
+import { feature } from '../../../lib/featureManagement';
+
+interface CardCoverProps extends CommonCardCoverProps {
+  imageProps: ImageProps;
+  videoProps?: Omit<VideoImageProps, 'imageProps'>;
+  isVideoType?: boolean;
+}
+
+export function CardCover({
+  imageProps,
+  videoProps,
+  isVideoType,
+  onShare,
+  post,
+}: CardCoverProps): ReactElement {
+  const { feedName } = useActiveFeedNameContext();
+  const [justUpvoted, setJustUpvoted] = useState(false);
+  const [hasInteracted, setHasInteracted] = useState(false);
+  const shareLoopsEnabled = useFeatureIsOn(feature.shareLoops);
```

Comment:
I had never used this `useFeatureIsOn`, is this what we use for boolean flags?

Reply:
Yes. Either enabled or disabled.

Reply:
It would also return `true` even if `shareLoops` is set to `false` for the user I think.